pandas/core/frame.py

```diff
@@ -261,6 +261,8 @@ def __init__(self, data=None, index=None, columns=None, dtype=None,
                 data = list(data)
             if len(data) > 0:
                 if is_list_like(data[0]) and getattr(data[0], 'ndim', 1) == 1:
+                    if hasattr(data[0], '_fields') and columns is None: # is namedtuple
```

Comment:
Let's also check for `isinstance(data[0], tuple)`. I can imagine lots of other classes with `_fields` attributes.